coalib/bears/requirements/DistributionRequirement.py

```diff
@@ -38,16 +38,18 @@ def install_command(self):
                         'redhat': 'yum',
                         'arch': 'pacman'}
 
-        if (platform.linux_distribution()[0] in manager_dict.keys() and
-                manager_dict[platform.linux_distribution()[0]]
+        if platform.system() == 'Linux':  # pragma: no cover
+            import distro
+            distro_name = distro.linux_distribution()[0]
```

Comment:
I think there should be an elif with a branch for mac and "anything unknown/windows", can you add brew support for mac while we're at it? (own commit of course :))